src/Lesson3/tasksForLesson_3_2.java

```diff
@@ -0,0 +1,51 @@
+package Lesson3;
+
+import java.util.Random;
+import java.util.Scanner;
+
+/**
+ * String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado", "broccoli", "carrot", "cherry", "garlic",
+ * "grape", "melon", "leak", "kiwi", "mango", "mushroom", "nut", "olive", "pea", "peanut", "pear", "pepper",
+ * "pineapple", "pumpkin", "potato"}.
+ * При запуске программы компьютер загадывает слово, запрашивает ответ у пользователя,
+ * сравнивает его с загаданным словом и сообщает, правильно ли ответил пользователь. Если слово не угадано,
+ * компьютер показывает буквы, которые стоят на своих местах.
+ * apple – загаданное
+ * apricot - ответ игрока
+ * ap############# (15 символов, чтобы пользователь не мог узнать длину слова)
+ * Для сравнения двух слов посимвольно можно пользоваться:
+ * String str = "apple";
+ * char a = str.charAt(0); - метод вернет char, который стоит в слове str на первой позиции
+ * Играем до тех пор, пока игрок не отгадает слово.
+ * Используем только маленькие буквы.
+ */
+public class tasksForLesson_3_2 {
+    public static void main(String[] args) {
+        String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado", "broccoli", "carrot", "cherry", "garlic",
+                "grape", "melon", "leak", "kiwi", "mango", "mushroom", "nut", "olive", "pea", "peanut", "pear", "pepper",
+                "pineapple", "pumpkin", "potato"};
+        Random rand = new Random();
+        int x = rand.nextInt(words.length - 1);
```

Comment:
```suggestion
        int x = rand.nextInt(words.length);
```
иначе пропускаем последнее слово (правая граница не включается в данном методе)